media.css

```diff
@@ -0,0 +1,364 @@
+@media screen and (min-width: 1140px) {
+  body {
+    width: 100%;
+  }
+
+  .menu-bar {
+    display: none;
+  }
+
+  .header {
+    background-image: none;
+  }
+
+  .nav-b2 {
+    background-color: #272a31;
+    display: block;
+    display: flex;
+    align-items: center;
+    color: #d3d3d3;
+    font-size: 12px;
+    gap: 20px;
+  }
+
+  .list2 {
+    font-family: 'Lato', sans-serif;
+    font-size: 12px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+  }
+
+  .test {
+    margin-left: 920px;
+  }
+
+  .list2 ul {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+    gap: 20px;
+  }
+
+  .list2 a {
+    text-decoration: none;
+    color: #d3d3d3;
+  }
+
+  .nav-b {
+    background-color: #d3d3d3;
+    display: block;
+  }
+
+  .nav-b .menuList {
+    display: flex;
+    gap: 40px;
+    margin-left: 520px;
+    font-size: 15px;
+    margin-top: -70px;
+  }
```

Comment:
- [OPTIONAL] Make sure the navigation links are centered vertically like in the design provided
Original Design 👇 
![Screenshot from 2022-07-29 15-35-52](https://user-images.githubusercontent.com/87965534/181760527-542910c9-9ef3-4e94-b234-1679623a2e5a.png)
Your design 👇 
![Screenshot from 2022-07-29 15-38-21](https://user-images.githubusercontent.com/87965534/181761928-9790d259-547a-4e8c-8ffa-98a5fc1958e5.png)